rbac/server/db/proposals_query.py

```diff
@@ -191,16 +191,17 @@ def fetch_open_proposals_by_opener(next_id):
     return resource
 
 
-def fetch_open_proposals_by_target(next_id):
+async def fetch_open_proposals_by_target(conn, next_id):
```

Comment:
I think we should fix our naming convention in another ticket if they're both interchangeable rather than use a new vague and possibly confusing name here

Reply:
Let's change to `target_id` then? Then in the docstring we can say a user or role UUID4 formatted ID. They aren't interchangeable, they're just the same UUID format.